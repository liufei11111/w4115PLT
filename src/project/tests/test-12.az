Int a;
Int b;
Int c;
Boolean b1;
Boolean b2;
Boolean b3;

a = 2;
b = 1;

c = (a + b) * 3;
c = (a + b + 3) * 2;
c = 2* (a + b);
c = a + (1 + a);
c = (1 + a) * (1 + b);
c = (a + b);

b1 = true;
b2 = false;
b3 = b1 && (b1 || b2);
b3 = (b1 && b1 && b1);

