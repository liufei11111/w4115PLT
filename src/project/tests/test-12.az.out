stmts:

int a;
int b;
int c;
bool b1;
bool b2;
bool b3;
a = 2;
b = 1;
c = ( a + b ) * 3;
c = ( a + b + 3 ) * 2;
c = 2 * ( a + b );
c = a + ( 1 + a );
c = ( 1 + a ) * ( 1 + b );
c = ( a + b );
b1 = True;
b2 = False;
b3 = b1 && ( b1 || b2 );
b3 = ( b1 && b1 && b1 );

fdecls:



